examples/visualize_clusters.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+from wellcomeml.ml.clustering import TextClustering
+from wellcomeml.viz import visualize_clusters
```

Comment:
I think there is an error here. Should be:

`from wellcomeml.viz.visualize_cluster import visualize_clusters`

A bit complicated, but it's because we are trying to access the module `visualize_clusters` inside `visualise_clusters.py`